src/main/java/mate/academy/webshop/controllers/ShoppingCartController.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.webshop.controllers;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.academy.webshop.lib.Injector;
+import mate.academy.webshop.model.ShoppingCart;
+import mate.academy.webshop.service.ShoppingCartService;
+
+public class ShoppingCartController extends HttpServlet {
+    private static final Long USER_ID = 1L;
+
+    private static final Injector injector = Injector.getInstance("mate.academy.webshop");
+    private final ShoppingCartService shoppingCartService =
+            (ShoppingCartService) injector.getInstance(ShoppingCartService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        ShoppingCart cart = shoppingCartService.getByUserId(USER_ID);
+        req.setAttribute("products", cart.getProducts());
+        req.setAttribute("userId", USER_ID);
```

Comment:
do we need to show that User_id to our User?(he will have access only for his own cart)

Reply:
Removed.